kpops/components/base_components/models/to_section.py

```diff
@@ -74,6 +76,23 @@ def extra_topic_role(cls, values):
             )
         return values
 
+    def __init__(
+        self, type: OutputTopicTypes | None = None, role: str | None = None, **kwargs
+    ):
+        type = self.__assign_type(type, role)
+        super().__init__(type=type, role=role, **kwargs)
+
+    @staticmethod
+    def __assign_type(
+        type_: OutputTopicTypes | None, role: str | None
+    ) -> OutputTopicTypes:
+        if type_ is None:
+            if role is None:
+                type_ = OutputTopicTypes.OUTPUT
+            else:
+                type_ = OutputTopicTypes.EXTRA
+        return type_
```

Comment:
please refactor this similar to FromSection

Reply:
done